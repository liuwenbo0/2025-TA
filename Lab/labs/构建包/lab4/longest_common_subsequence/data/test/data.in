10
FHIAFHJADGI
AGCAHBGGKKDDFKAF
IKJEEDAHFJJGCF
GIGEBAFECK
JHDABIFKCUJLBSNITGSLEPTSNMOJRANB
INLAUAHQLSFQOLDCSAJCSSIDUEISLJEKJPDBEEBLQGJTC
HEGTPJBTTFCJMTOBBQLAMHHNKLKHRPMCAEDOFTBAUNSIFSREN
TABSFOTSMNKBOFSDIUNOBQJAEKOIFUFGGNPQTGL
DEADHCLLGFHGDBPDAHIBCFCKHBC
TNOLROUNQMDJLPTSIRDIKGSNALTSURU
LCMRJDJPHACHJAPJCGQLHAARCDECMEHSLGALGFJOT
OMTJNUQLNOCQLUDJUSKOKLHOFULITMPULI
HTJJOPMHUSGQQEFEDONDLBEDTOQQCERBBUFHLCCDTGLMLLPPAGPMJFTLJQUNPGEGHENLEKDOKNADOOOMMKADTF
MTPRQRQQUUIHARRKFDLPTQNMKPHKCEPPARSCLRTOJGRGBAJHLCTHLQDGQLBCDEOHANQB
MIEFIDAOMBPDRAJMBMMHDIPPLACJDCCAEOEHSHEBAHAGRJJCJIBADKANADDLQAMARHFEDFOHOKGEMNOPJENNDCEPC
MSMTOSTTRNDIRLUHPORKJIMUIRRRSSCRNMJJFNEUNIRMLSUTSLUJPOTMTRP
QGCKRBQSHAJDOURQTNSSHEQUAKQOMCUEBNTKSDDHAMEUNGGBCGTMMDJCMMASIUUQGCTLSUPPDJUKSCNDQDHEKOLQPKAIRSJMNHMNBRQUGAHIBAQSFDOKUHHUFEHOIEFTPIOPNFGODLQIBMTKOQEQGJKGUUNAPHMJUFGHLLNAJGFAFKEIDCQQBJLKILEJJOGUALOABFHLARIMLPTNBBKFEMFMEFBAOQRHIAHOAPLCKATTJNPARPSGQAJEUSDOKPHMGTBSTKQPRKAIRCMGSDUSMJGKAQCMJTSPMJEDLKCJEATJJ
GBIKMUCPQBTDODRRQHOGOBDFRASHBFUOCIDTHQLAHNNAKDDDPOTJRLTTPFJGPMLBDSKPHEHPSCDTMJTFICCRSHJNMOLKSSCNBONBEPLBARHTGQSHKHFKERDSPDMCLTFJOQICNTTOKCHONSFUGOOKANOLFRLFCMIAEIRUUEUINJHOHTIKQDLBRIIEPCSOBBLAKQPBHUCUSLPPCHTEAHOURGOGQHJPQSUTOQRBRBBOPOFOPDIPCPUQKKTQETCOKKUOFKMFQSEPLOLCLUGJTBNLMGTBRSGTPSRDIUCGJJKSBEGNBDHIGRGGUMQGSLFDRDMKJCGHFEECFFKFTBBHMCOTBUNURLUITFOIQUCCEKEGPPOMIJPSMMDDKQFGGGGBIOHERSSMUKNIKLNLTPLHQJAERRALGAPJEMOILIREGHLQGKHCDOOGBHIOCJGRFTIICRJNHOBMPUIUDNKBBJATEUSAIEUEUQUPTNCENJOSQAOSUFM
EEICJTXKKVXCXJOWRCVLTTSQHWKBBMIGTQLOVJGVIYZGCQJVPVOTWUCSETIDICYHTCMAJPHXYYOOEOVUXVUPLOKLPBOVQDWDYPBXGAJUQWADGEEDJKGURHSXDVYLMYFJQLWZLDROUYLQOBSGEMWDOIBQVCYEDFVQOPFHKQMHUSXQACOOGJXCOXBFZLWCXFVQAAVFEGKCIRQQGDYHLJMAQZQIFUAOUBUKYPAVEBVDUJGSULAHKPRFPNZQAQGVFDXWTQFLCEILPMSZIZFWBONXPLCYQITTPKPBCFSESGFBIQNIPOLEFRELPHJTHQRZSPHBNUMGRIFMWIZTFUHQIBGAXDVYSYVGXLSPMINYKBYUMEPUBRXUOAVYOVDBIELZDOBGQCJZNJBEXALKGHYWIOXZBVXZFCSHCOZMEFWCRVYIBJDFQVQMHXDPCCJODLGVKPLRFDEDPZOPRFEECHWSZHVCDOOEJLCHCWCEKTFRDMOWHSUEAVBRAWMIHZSNFHRAFBQEAWDIXZNPPFWIEAIVTMPQZQTSVNVWJMGVHUESSSAXGMOYWDGVWIOUZUQDEBIJCQYCFTAFTUWTGXAVEMCIUQMXENPRPMYZREXSHNVTESSTWHYTMATXBUZXPSTPYGFXPHPFCKJDBFBXIMEESWYNDFHOMCNWJTFRUVZWBHLZBVEBYEUCEPGCDPMGHCIBFXGRVNACXWLOPXSVZCLWYMWDMOLJTAGUWUVHXIFDJQSHLQAUEGLQRBXUWPOZFVEGDPVNWMEDXBMXAGIMXMXGJLNNFKWBRQSHXFCANWPOFLQEJAFDKTNLPGVLEXCYGLRZBKHCMXVUOBYXCXZXQMLSWVLMBVCSDTEFCQTMLILWZTBSCFHJTCYTONMLIIVRKBBLVNMHIPRGERYRPJPJYKCHNEYFYMIEFYMAJRNWJSLYTBXMDUWTFZAFKXYFMCNLJSEBZLEYGEGUUJMKYYLFMMVVILJWGCYHRDLD
KZFZGTVMOWLBEQTJSODVHBUUMEXPPCDXQHCEQLDFAVGQLVITUZZNGJEOTBKJZCRTGGLUIAMNCORBKIWDOZITQGQIPHVWUIJWHSMGOGXOMLUQURCWOGGHHJJCEKGGLLAMQGXHCFCMRAKGROJGXIZWXASCASINZXCDXEWCULOFITYLHXHZXMJDLISUPDYLSHAHCIPZOPTULLYISXSCZJQAWAGNNGCGGIWKHNARJISVCDPZGGTLLUDVCSJXEUBFGYJDIMTPFTGYTKFNTYLNFLBFUTSYOVXMLIGOAIQZLXWFRSLJUCDXSSAMGCIFDLJASHCJJWHLKQNIPXZJGLPTEDZXMTHVZGRUHHCKUFDECAKRRZOQFQCDUZHJSDASKQGLQLNNBKUUYTQTHFCJRAAWTBCCURPZPUMCSMQHEBHQRFIGAMBLZXYCVEVLUZCUQXVRJGEKYAOFVRVCOHLDAVKJIYKHCCMNJPDHKKIMPSMVCYQXLGWLBHLDKJLZWTHKFNIZEBVLGOZRNWWVXDCKGNCKQTKQIDHOOCIWRPKXESLDTGXTAURGLBGMPYQLPZBNSBZHANBIYPUZIQURWRBRBXMHBRDPDRERBOBAJULDLWRPKRTFBLSPNKX
