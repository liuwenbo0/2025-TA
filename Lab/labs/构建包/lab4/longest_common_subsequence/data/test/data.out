3
5
11
14
6
10
25
17
124
251
